Make codeblock fold/unfold use a larger clickable area

ppwwyyxx opened the pull request:
Before this change, for codeblocks with folding enabled, users have to click the small arrow to toggle fold/unfold:
![2022-05-07_14-09](https://user-images.githubusercontent.com/1381301/167271797-ee82420d-1911-4175-92b3-2c5801e6d7ad.png)

I find it very difficult to click since it's too small.

After this PR, clicking anywhere in the div below would toggle fold/unfold:
![2022-05-07_15-38](https://user-images.githubusercontent.com/1381301/167274065-c78acd9c-e6ab-4115-867c-41610a97e9cc.png)

The hyperlink and copy button on the right is unaffected.

Also changed the cursor style to pointer, for this div.

When folding is disabled, the cursor style stays unchanged - this is a use case of https://github.com/ppoffice/hexo-theme-icarus/pull/1071


This is only my personal opinion, so feel free to dismiss it, and I can keep it in my own customization.

ppoffice commented:
@ppwwyyxx Will the hyperlink in the header section and the copy code button still work?
![image](https://user-images.githubusercontent.com/8849362/167272389-0bdc9bdf-7b3c-48d0-97bd-c6a90068e04a.png)

ppoffice commented on source/js/main.js:
```diff
@@ -111,7 +111,7 @@
             });
 
-            $('figure.highlight figcaption .fold').click(function() {
+            $('figure.highlight figcaption').click(function() {
```
Does this mean clicking the caption will always fold or unfold the code block even if folding is disabled in the `_config.yml`?

ppwwyyxx replied:
No, this javascript is inside a `if (fold)` branch, so when folding is disabled in config, this line will not execute.

ppwwyyxx commented:
Thanks for your feedback. In order to keep hyperlink and copy button working, I changed the toggle area to be the filename only. I updated the PR description.
